src/pages/SendNft.vue

```diff
@@ -250,6 +297,45 @@ function handleSetGasPrice(value) {
   gas.value = value
 }
 
+async function getMVXTransactionObject() {
+  const accountHandler =
+    getRequestHandler().getAccountHandler() as MultiversXAccountHandler
+
+  const factory = new TransferTransactionsFactory(new GasEstimator())
+  const transfer = TokenTransfer.nonFungible(
+    props.collectionName as string,
+    props.nonce as number
+  )
+
+  return factory.createESDTNFTTransfer({
+    tokenTransfer: transfer,
+    nonce: await accountHandler.getAccountNonce(),
+    sender: new Address(userStore.walletAddress),
+    destination: new Address(recipientWalletAddress.value),
+    chainID: MVXChainIdMap[rpcStore.selectedChainId as number],
+  })
```

Comment:
Move this to MVXAccountHandler

Reply:
[done](https://github.com/arcana-network/wallet-ui/pull/615/commits/9c1bd413024ae7b894e80d5e3f1849e9fb01cb87)